### PR TITLE
feat: add support for computed_members

### PR DESCRIPTION
**Pull request by waxb**
- chore: changelog updated
- feat: add support for `computed_members`

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -33,7 +33,7 @@ resource "google_secret_manager_secret" "secret" {
               for_each = try(replicas.value.customer_managed_encryption, [])
 
               content {
-                kms_key_name = customer_managed_encryption.value.kms_key_name
+                kms_key_name = customer_managed_encryption.value
```
Comment (mariux) [inline main.tf:36]: looks like breaking the functionality?

**Review by mariux — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (waxb) [reply main.tf:36]: it was broken before

File: main.tf [hunk 2]
```diff
@@ -33,7 +33,7 @@ resource "google_secret_manager_secret" "secret" {
               for_each = try(replicas.value.customer_managed_encryption, [])
```
Comment (mariux) [inline main.tf:33]:
```suggestion
              for_each = try([replicas.value.customer_managed_encryption], [])
```

**Review by mariux — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (mariux) [inline main.tf:36]:
```suggestion
                kms_key_name = customer_managed_encryption.value.kms_key_name
```

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**